From 6625e563d600eba34cf2c71df7572ad1c421c6fd Mon Sep 17 00:00:00 2001
From: Dmitry Baryshkov <dmitry.baryshkov@oss.qualcomm.com>
Date: Thu, 18 Sep 2025 05:37:13 +0300
Subject: [PATCH] qcom: move LEMANS EVK firmware to correct location

All firmware for the Lemans platform is located under qcom/sa8775p/
subdir. Move audio topology for Lemans EVK to the same location and
provide (ugh) backwards-compatible symlink.

Cc: Mohammad Rafi Shaik <mohammad.rafi.shaik@oss.qualcomm.com>
Signed-off-by: Dmitry Baryshkov <dmitry.baryshkov@oss.qualcomm.com>
Upstream-Status: Backport [https://gitlab.com/kernel-firmware/linux-firmware/-/commit/6625e563d600eba34cf2c71df7572ad1c421c6fd]
---
 WHENCE                                        |   3 ++-
 qcom/{qcs9100 => sa8775p}/LEMANS-EVK-tplg.bin | Bin
 2 files changed, 2 insertions(+), 1 deletion(-)
 rename qcom/{qcs9100 => sa8775p}/LEMANS-EVK-tplg.bin (100%)

diff --git a/WHENCE b/WHENCE
index a93effa5008b..e22d8218959f 100644
--- a/WHENCE
+++ b/WHENCE
@@ -8469,7 +8469,8 @@ Originates from https://github.com/linux-msm/audioreach-topology.git
 --------------------------------------------------------------------------
 
 Driver: qcom-qcs9100 - Qualcomm ASoC topology
-File: qcom/qcs9100/LEMANS-EVK-tplg.bin
+File: qcom/sa8775p/LEMANS-EVK-tplg.bin
+Link: qcom/qcs9100/LEMANS-EVK-tplg.bin -> ../sa8775p/LEMANS-EVK-tplg.bin
 Version: v0.1.0
 
 Licence: Redistributable. See LICENCE.linaro for details
diff --git a/qcom/qcs9100/LEMANS-EVK-tplg.bin b/qcom/sa8775p/LEMANS-EVK-tplg.bin
similarity index 100%
rename from qcom/qcs9100/LEMANS-EVK-tplg.bin
rename to qcom/sa8775p/LEMANS-EVK-tplg.bin
-- 
2.51.0

